xc7/utils/prjxray_routing_import.py

```diff
@@ -230,6 +231,28 @@ def check_feature(feature):
 
         return ' '.join((feature, enable_buffer_feature))
 
+    if CLK_HROW_CK_MUX_REGEX.fullmatch(feature_path[-1]):
+        bufhce_loc_string = feature_path[-1].split('_')[-1]
```

Comment:
Because you already have a regular expression, please add capture groups to the `CLK_HROW_CK_MUX_REGEX` and use those capture groups, instead of ad-hoc string slicing.

Reply:
Done